components/admin/lessons/AdminLessonInfo.tsx

```diff
@@ -152,14 +153,15 @@ export const AdminLessonInfo: React.FC<LessonInfoProps> = ({
   selectedLesson
 }) => {
   // true when user clicks on `create new lesson` button
-  if (lessons && selectedLesson === lessons.length) {
+  if (!lessons || selectedLesson === lessons.length) {
     return <NewLesson setLessons={setLessons} />
   }
 
   // set currently selected lesson
-  const lesson = lessons && lessons[selectedLesson]
+  const lesson = lessons[selectedLesson] ? lessons[selectedLesson] : lessons[0]
```

Comment:
Incorrect lesson id case.

Reply:
This can be shortened to 
```
const lesson = lessons[selectedLesson] || lessons[0]
```
This will make the code shorter and easier to understand. In this scenario, it makes more contextual sense to use the logical OR operator rather than ternary. Your use of ternary operator is just doing what the logical OR operator already does.